### PR TITLE
feat(ui): add notification when copying urn

### LINKED ISSUE #751: Add a notification when you click the copy button
Text should be "Radicle ID for **project-name** copied to your clipboard."

### PR DESCRIPTION
**Pull request by juliendonck**
Adds a optional notification to `<Copyable />` and adds those where we use `<Urn />`

closes #751

### CODE REVIEW

File: ui/DesignSystem/Component/AdditionalActionsDropdown.svelte [hunk 1]
```diff
@@ -121,7 +121,7 @@
     <div out:fade={{ duration: 100 }} class="modal" hidden={!expanded}>
       {#if headerTitle}
         <div class="header">
-          <Urn urn={headerTitle} />
+          <Urn urn={headerTitle} notificationTxt="Copied to your clipboard" />
```
Comment (rudolfs) [inline ui/DesignSystem/Component/AdditionalActionsDropdown.svelte:124]: Could we call the prop `notificationText`?

File: ui/DesignSystem/Component/Copyable.svelte [hunk 2]
```diff
@@ -23,6 +25,8 @@
       copyContent !== null ? copyContent : slotContent.textContent;
     copyToClipboard(content.trim());
 
+    notificationTxt ? notification.info(notificationTxt) : null;
```
Comment (rudolfs) [inline ui/DesignSystem/Component/Copyable.svelte:28]:
```suggestion
if(notificationTxt) {
  notification.info(notificationTxt);
}
```

The ternary operator feels a bit weird here.

File: ui/DesignSystem/Component/Header/Large.svelte [hunk 3]
```diff
@@ -10,6 +10,15 @@
...
   export let variant = null; // profile | org
 
+  const name =
+    variant === "profile" && entity.registered
+      ? entity.registered
+      : variant === "profile" && !entity.registered
+      ? entity.metadata.handle
+      : variant === "org"
+      ? entity.id
+      : null;
```
Comment (rudolfs) [inline ui/DesignSystem/Component/Header/Large.svelte:20]: I feel like nested if's would be more legible here.

Comment (juliendonck) [reply ui/DesignSystem/Component/Header/Large.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006

File: ui/DesignSystem/Component/Copyable.svelte (hunk 2 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/Copyable.svelte]: removed this and gave it a default text -> https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006

File: ui/DesignSystem/Component/AdditionalActionsDropdown.svelte (hunk 1 above)
Comment (juliendonck) [reply ui/DesignSystem/Component/AdditionalActionsDropdown.svelte]: https://github.com/radicle-dev/radicle-upstream/pull/758/commits/0b9867696a1ef5ce4d154ef1d30462fa65af6006

**Pull request merged**